ansible/roles/test/files/ptftests/arista.py

```diff
@@ -58,27 +58,27 @@ def connect(self):
         self.shell = self.conn.invoke_shell()
 
         first_prompt = self.do_cmd(None, prompt = '>')
-        self.hostname = self.extract_hostname(first_prompt)
+        self.arista_prompt = self.get_arista_prompt(first_prompt)
 
         self.do_cmd('enable')
         self.do_cmd('terminal length 0')
 
         return self.shell
 
-    def extract_hostname(self, first_prompt):
+    def get_arista_prompt(self, first_prompt):
         lines = first_prompt.split('\n')
         prompt = lines[-1]
-        return prompt.strip().replace('>', '#')
+        return prompt.strip().replace('>', '.*#')
 
```

Comment:
What is the purpose of adding '.*'?

Reply:
To accommodate 'config' mode on the Arista device. The prompt changes at each hierarchy
ARISTA(config)#int po1
ARISTA(config-if-Po1)#